app-backup/rear/rear-2.6.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit udev
+
+DESCRIPTION="Relax-and-Recover is a setup-and-forget bare metal disaster recovery solution"
+HOMEPAGE="http://relax-and-recover.org/"
```

Comment:
Can you add the Github page here as 2ndary HOMEPAGE?

Reply:
@juippis yes, both done